tests/test_keep_alive_timeout.py

```diff
@@ -8,10 +8,11 @@
 import httpx
```

Comment:
Can we remove httpx from here?

Reply:
Maybe at some point, but I don't think we are there yet. We really need a new way to test timeouts. I think I have an idea, but there are higher priorities that are on my list to accomplish before the next release.

With that said, if anyone wants to take on these tests, I would be happy to help.

Reply:
I don't see a problem in using third-party requirements on our testing, since they are actually required in our own testing package, unless we remove `httpx` from `sanic_testing` in the future.

Reply:
I meant that I really don't like the tests themselves that require us to hack httpx here. I have no intention of removing httpx from the testing client because the ASGI reach in is awesome.